papyri/directives.py

```diff
@@ -0,0 +1,72 @@
+"""
+Various directive handlers.
+"""
```

Comment:
I'm actually unsure we want the handler to be simply function, 
we might want to have class/methods, so that we can pass in configuration to the class, and or have it keep state.